tree/ntuple/v7/inc/ROOT/RNTupleUtil.hxx

```diff
@@ -126,8 +126,6 @@ constexpr NTupleSize_t kInvalidNTupleIndex = std::uint64_t(-1);
 using DescriptorId_t = std::uint64_t;
 constexpr DescriptorId_t kInvalidDescriptorId = std::uint64_t(-1);
 
-namespace Experimental {
-
 /// Addresses a column element or field item relative to a particular cluster, instead of a global NTupleSize_t index
 class RNTupleLocalIndex {
```

Comment:
We are missing a `using` declaration - or is this not needed because we recently renamed the class anyway?

Reply:
I didn't include usings for this and the following class because while public, it should be very rare use. I was planning to add usings only for the common types. We anyway don't cover everything (e.g. constants like `kInvalidDescriptorId`). But this could be handled differently, too.

In general, the problem with usings is that we clutter compilation with deprecation warnings from all classes that are still in experimental, because they pick up the `ROOT::Experimental::Xyz` version then. For the typedefs, I felt it makes anyway sense to universally prefix them by `ROOT::`. But I wouldn't necessarily want that for all classes.